src/PluginRegistrationDialog.cpp

```diff
@@ -514,6 +506,7 @@ void PluginRegistrationDialog::PopulateOrExchange(ShuttleGui &S)
 //      S.EndStatic();
       S.EndVerticalLay();
 
+      S.Id(ID_RescanPlugins).AddButton(XXO("&Rescan plugins"));
```

Comment:
Does the hot key &R conflict with another control in this dialog?  Answer: no.  But I asked the question and checked!

Reply:
I built and ran and looked.  This button looks strange on a row by itself in the middle of a wide dialog.  Maybe it should align right.